.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+
+
+
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "30 1 * * *""
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 60
+          exempt-issue-labels: "Blocked"
+          days-before-issue-close: 7
```

Comment:
Just a personal opinion but perhaps this should be closer to `14` days just to account for long holiday breaks and vacations

Reply:
👍